python/darpinstances/instance.py

```diff
@@ -179,6 +178,51 @@ def load_vehicles(vehicles_path: str) -> List[Vehicle]:
     return vehicles
 
 
+equipment_list = [
```

Comment:
This should be implemented as an Enum (see for example, Action_type in instance_objects.py)

Reply:
I have it as a list to be able to iterate through it, why should it be an enum?